client/src/components/continuousLegend/index.js

```diff
@@ -128,7 +135,16 @@ class ContinuousLegend extends React.Component {
   }
 
   render() {
-    const { colorAccessor } = this.props;
+    const { colorAccessor, colorScale } = this.props;
+
+    if (
+      colorScale &&
+      colorScale.domain &&
```

Comment:
You can use optional chaining here:
```suggestion
      colorScale?.domain &&
```